src/apprt/gtk/Window.zig

```diff
@@ -247,9 +248,94 @@ pub fn init(self: *Window, app: *App) !void {
         }
         c.gtk_widget_add_css_class(@ptrCast(gtk_window), "devel");
         c.gtk_widget_add_css_class(@ptrCast(warning_box), "background");
+        c.gtk_widget_add_css_class(@ptrCast(warning_box), "banner-box");
         c.gtk_box_append(@ptrCast(box), warning_box);
     }
 
+    // Display a banner informing about crash reports if any exist
+    {
+        var crash_dir = try crash.defaultDir(self.app.core_app.alloc);
+        defer self.app.core_app.alloc.free(crash_dir.path);
+        var it = try crash_dir.iterator();
+        defer it.deinit();
+
+        if (try it.next()) |_| {
+            const warning_text = try std.fmt.allocPrintZ(
+                self.app.core_app.alloc,
+            \\ There are existing crash reports located at <a href="file://{s}">{s}</a>.
+                 \\ If possible, <a href="https://github.com/ghostty-org/ghostty?tab=readme-ov-file#crash-reports">report them to the developers</a> to help improve the application.
+                    ,
```

Comment:
It's wonderful that clicking the links takes you someplace useful. I was thinking that we'd immediately be inundated with requests to get some kind of GUI to manage crash reports (and we may still need that eventually) but clicking the link brings up Gnome's file manager with the crash reports directory so it's pretty easy for users to manage their crash reports.